bin/cdklocal

```diff
@@ -79,6 +81,14 @@ getMethods(CdkToolkit.prototype).forEach(meth => {
   }
 });
 
+Object.defineProperty(ToolkitInfo.prototype, 'bucketUrl', {
+  get() {
+    const bucket = this.requireOutput(bootstrap.BUCKET_NAME_OUTPUT);
+    const domain = this.requireOutput(bootstrap.BUCKET_DOMAIN_NAME_OUTPUT);
+    return `https://${domain.replace(bucket, '')}/${bucket}`
```

Comment:
Looks great, thanks for this PR @rehmanvirani-10p ! Can you please share a small example of which `bucketUrl` gets generated - prior to and after your change..?

Wondering if we should replace the bucket name including the dot (`.`) in the host name, e.g., `mybucket.example.com` to `example.com` (instead of `.example.com`)

Reply:
@whummer You are absolutely right. The example generated URL is `https://.s3.us-east-1.amazonaws.com/cdktoolkit-stagingbucket-1c1bc7c1`. Updating the code accordingly.

Reply:
Fixed.